core/arch/arm/sta/sub.mk

```diff
@@ -1,5 +1,6 @@
 srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += sta_self_tests.c
 srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += core_self_tests.c
+srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += sta_debug.c
```

Comment:
`debug.c` is a too generic name to my mind. What about changing it completely, something like `optee_os_status` as I intend to add stats on allocation too? @jenswi-linaro what do you think?